src/Core/Utilities/PasswordManagerPlanStore.cs

```diff
@@ -124,51 +237,80 @@ public static IEnumerable<Plan> CreatePlan()
             },
             new Plan
             {
-                Type = PlanType.EnterpriseAnnually2019,
-                Name = "Enterprise (Annually) 2019",
-                IsAnnual = true,
-                Product = ProductType.Enterprise,
+                Type = PlanType.TeamsMonthly2020,
```

Comment:
You are changing the EnterpriseAnnually2019 to TeamsMonthly2020. I thought we should be changing  Enterprise to  Enterprise not Teams. Please confirm that this change is in order

Reply:
I could be wrong, but I think this is just a weird diff because some of the plans were reordered. The `EnterpriseAnnually2019` plan is here now: https://github.com/bitwarden/server/blob/49214fb9e3406a4faf656ada1e363c5d66c6c1bb/src/Core/Utilities/PasswordManagerPlanStore.cs#L309

Reply:
Sorry about that, yes this is because I had reordered the plans. I just pushed a commit that restored the original ordering, so the diff should be more clear. 2019 plans are still there, the only thing I've changed for those plans is the features that are enabled for them